### ISSUE TITLE
Image-to-text doesn't seem to work 

### ISSUE THREAD

**Issue by patrickvonplaten** [SHI-Labs/Versatile-Diffusion#5]
Hey @xingqian2018,

Great work! 

I currently have problems when running the image-to-text application. 

When doing the following:

and then running the following command:
```
python inference.py --app image-to-text --model official --fp16 --image assets/boy_and_girl.jpg --nsample 1 --seed 0
```

I'm getting as an output:
```
viewer surveys the fresh local produce
```

I'm running on 1.12.1+cu113 on a V100

**Comment by xingqian2018** [comment]
I think the issue is on the latest dev version of the `transformers` library. The issue will be resolved when using `transformers==4.24.0`

**Comment by patrickvonplaten** [comment]
For reference, this issue will be solved for `transformers>4.24.0` as soon as this issue is resolved: https://github.com/huggingface/transformers/issues/20394
